### PR TITLE
AG-12333 Update font styles

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-12333

### CODE REVIEW

**Review by alantreadway — APPROVED** [review]
LGTM, looks like this is just applying previous design feedback for the most part.

File: packages/ag-charts-community/src/chart/dom/theme.css [hunk 1]
```diff
@@ -58,6 +61,7 @@
         var(--ag-charts-toolbar-foreground-color) 6%
     );
     --ag-charts-toolbar-gap: var(--ag-charts-size);
+    --ag-charts-toolbar-font-family: var(--ag-charts-font-family);
```
Comment (markdurrant) [inline packages/ag-charts-community/src/chart/dom/theme.css]: I would remove this as a separate variable, and have a single place to control font-family across all charts. 

(I can't see a use case for having a different font-family for toolbars to the rest of charts).

File: packages/ag-charts-community/src/chart/dom/theme.css [hunk 2]
```diff
@@ -70,6 +74,7 @@
     --ag-charts-popover-focus-color: var(--ag-charts-toolbar-focus-color);
     --ag-charts-popover-hover-color: var(--ag-charts-toolbar-hover-color);
     --ag-charts-popover-active-background-color: var(--ag-charts-toolbar-active-background-color);
+    --ag-charts-popover-font-family: var(--ag-charts-toolbar-font-family);
```
Comment (markdurrant) [inline packages/ag-charts-community/src/chart/dom/theme.css]: I would remove this as a separate variable, and have a single place to control font-family across all charts. 

(I can't see a use case for having a different font-family for toolbars to the rest of charts).

**Review by markdurrant — CHANGES_REQUESTED** [review]
I would remove the specific variables for setting a font family for toolbar or popover. Can't see a use case for this. 

Otherwise everything looking good.

File: packages/ag-charts-community/src/chart/dom/theme.css (hunk 2 above)
Comment (jacobp100) [reply packages/ag-charts-community/src/chart/dom/theme.css]: Updated

**Review by markdurrant — APPROVED** [review]
LGTM 👍

**Pull request merged**